rasa/server.py

```diff
@@ -235,7 +245,7 @@ async def get_tracker(
     _validate_tracker(tracker, conversation_id)
 
     # `_validate_tracker` ensures we can't return `None` so `Optional` is not needed
-    return tracker  # pytype: disable=bad-return-type
+    return cast(DialogueStateTracker, tracker)
```

Comment:
Rick is currently removing this in https://github.com/RasaHQ/rasa/pull/6979

Reply:
well, in that case, we'll simply handle the merge in either of the PRs, depending on which one gets merged first :)